app/views/posts/show.html.erb

```diff
@@ -1,5 +1,9 @@
 <%= render partial: "post_presenter_show" %>
-<%= render partial: "shared/comment/form_new", locals: { comments_parent: post, url: (post_comments_path(post)), submit_text: "Comment On Post" }  %>
+<%= render partial: "shared/comment/form_new",
```

Comment:
can this template live under `app/views/comments`?

Reply:
It could. I like having it under shared because both the Posts and Comments controllers are using it, but if having a shared folder is seen as an excess, then I'd want to move both comment partials to `views/comments`, and then move everything from `shared/votes` to a `votes` dir. My thinking was that any partials that are being used by another controller would go in the `shared` dir.

Reply:
I can dig that line of thinking 👍. I just figured the form is related to comments, so it should live under the comments dir.